docroot/modules/custom/yukon_w3_custom/yukon_w3_custom.module

```diff
@@ -0,0 +1,27 @@
+<?php
+
+/**
+ * @file
+ * Yukon w3 migrate module.
+ */
+
+use Drupal\Core\Form\FormStateInterface;
+
+/**
+ * Implements hook_theme().
+ */
+function yukon_w3_custom_theme() {
+  return [
+    'edit_language' => [
+      'variables' => ['items' => NULL, 'language' => NULL],
+      'template' => 'edit-language',
+    ],
+  ];
+}
+
+/**
+ * Implements hook_form_alter().
+ */
+function yukon_w3_custom_form_alter(&$form, FormStateInterface $form_state, $form_id) {
+  $form['#attached']['library'][] = 'yukon_w3_custom/yukon_w3_custom.css';
```

Comment:
I doubt that we need this for all forms on the site. Please attach it only to forms where it should be useful.

Reply:
On D7 it is for all admin side buttons. Thats why we implemented it globally. Is there anything, that we need to consider here.

Reply:
Right. but the issue is that this CSS applies to any form on the side, not only admin forms.
Try to add extra check if it is admin form, something like
`if (\Drupal::service('router.admin_context')->isAdminRoute()) {
 // do stuff
}`